src/NexusMods.App.UI/Pages/Settings/SettingsView.axaml

```diff
@@ -82,7 +82,7 @@
 
             <Border Grid.Row="2"
                     x:Name="FooterBorder"
-                    Classes="Footer">
+                    Classes="Footer OutlineWeak">
```

Comment:
You are missing `BorderThickness="0,1,0,0"`. The outline should appear only at the top, to distinguish it from settings entries that have the same color.